ci/scripts/r_test.sh

```diff
@@ -90,11 +90,32 @@ SCRIPT="as_cran <- !identical(tolower(Sys.getenv('NOT_CRAN')), 'true')
       message('Running minio for S3 tests (if build supports them)')
       minio_dir <- tempfile()
       dir.create(minio_dir)
-      pid <- sys::exec_background('minio', c('server', minio_dir))
-      on.exit(tools::pskill(pid))
+      pid_minio <- sys::exec_background('minio', c('server', minio_dir))
+      on.exit(tools::pskill(pid_minio), add = TRUE)
     }
   }
 
+  if(reticulate::py_module_available('pyarrow')){
+    old_pp <- Sys.getenv('PYTHONPATH', unset = NA)
+    Sys.setenv(PYTHONPATH = './inst')
```

Comment:
Prepending the python path does not work with `sys::exec_background`  so we have to manually set and re/unset the python path as not to interfere with any of the following tests etc..
This would not be necessary with `reticulate::import_from_path`

Reply:
I see. This is surely not very nice and requires manual cleanup that I guess can go wrong in case of crashes. One possible alternative is do perform the path addition in python itself:

```
 python -c "__import__(\"sys\").path.append(\"./inst\"); __import__(\"demo_flight_server\").DemoFlightServer(port=8089).serve()"
```

Reply:
Yep that works!